standalone-metastore/metastore-common/src/main/thrift/hive_metastore.thrift

```diff
@@ -1752,13 +1763,28 @@ struct ExtendedTableInfo {
  4: optional list<string> requiredWriteCapabilities // capabilities required for write access
 }
 
+struct DropTableRequest {
+ 1: optional string catalogName,
+ 2: required string dbName,
+ 3: required string tableName,
+ 4: optional bool deleteData,
+ 5: optional EnvironmentContext envContext,
+ 6: optional bool dropPartitions,
+ 7: optional string indexName
```

Comment:
what's the `indexName` for?

Reply:
This is not required. Removing it the next patch